internal/config/config_test.go

```diff
@@ -22,6 +23,26 @@ func TestLoadConfigEnv(t *testing.T) {
 	assert.Zero(t, config.Home, "Shouldn't set Home property when loading config")
 }
 
+func TestLoadConfigEnv_WithForcePrependEnv(t *testing.T) {
+	t.Run("When ASDF_FORCE_PREPEND env given yes", func(t *testing.T) {
+		os.Setenv("ASDF_FORCE_PREPEND", "yes")
+		defer os.Unsetenv("ASDF_FORCE_PREPEND")
+
+		config, _ := loadConfigEnv()
+
+		assert.True(t, config.ForcePrepend, "Then ForcePrepend property is true")
+	})
+
+	t.Run("When ASDF_FORCE_PREPEND env given any string other than yes", func(t *testing.T) {
+		os.Setenv("ASDF_FORCE_PREPEND", "no")
+		defer os.Unsetenv("ASDF_FORCE_PREPEND")
```

Comment:
Same comment here.

Reply:
It is the same as https://github.com/asdf-vm/asdf/pull/2011#discussion_r1991408214.